hw1-band-matrices/README.md

```diff
@@ -0,0 +1 @@
+# Homework 1 - Band matrices
```

Comment:
It would be nice to add some content here. It doesn't need to be a lot. A short description what band matrix is and a simple example.